docs/docs/architecture/attestation.md

```diff
@@ -186,6 +186,40 @@ The latter means that the value can be generated offline and compared to the one
 | 16&ndash;23 | Unused                                                           | -                             | -                           |
 
 </tabItem>
+</tabItem>
+<tabItem value="aws" label="AWS">
+
+Constellation uses the [vTPM](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nitrotpm.html) (NitroTPM) feature of the [AWS Nitro System](http://aws.amazon.com/ec2/nitro/) on AWS for runtime measurements.
+
+The vTPM adheres to the [TPM 2.0](https://trustedcomputinggroup.org/resource/tpm-library-specification/) specification.
+It provides [UEFI secure boot](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/uefi-secure-boot.html) that's based on the secure boot process feature of Amazon EC2.
```

Comment:
I think we need to refer to measured boot instead of secure boot here, right @daniel-weisse?

Reply:
Yes, measured boot should be the more important feature.
We can talk about secure boot, but I think we shouldn't refer to the implementation by AWS, since our boot chain probably works differently.

Maybe @malt3 can provide some input for the secure boot section?